src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -575,22 +643,26 @@ def getJobStatus(self, jobIDList):
 
         # Renew jobs to be renewed
         # Does not work at present - wait for a new release of ARC CEs for this.
-        result = self._renewJobs(jobsToRenew)
-        if not result["OK"]:
-            return result
+        if jobsToRenew:
+            result = self._renewJobs(jobsToRenew)
+            if not result["OK"]:
+                return result
 
         # Kill jobs to be killed
-        result = self.killJob(jobsToCancel)
-        if not result["OK"]:
-            return result
+        if jobsToCancel:
+            result = self.killJob(jobsToCancel)
+            if not result["OK"]:
+                return result
 
         return S_OK(resultDict)
 
     #############################################################################
 
-    def getJobOutput(self, jobID, _localDir=None):
-        """Get the specified job standard output and error files.
-        The output is returned as strings.
+    def getJobLog(self, jobID):
+        """Get pilot job logging info
+
+        :param str jobID: pilot job identifier
```

Comment:
I have decided to streamline these variable names such as:
- `arcJobID`: refers to the ARC JobID format `12345678`
- `jobID`: refers to the DIRAC JobID format `https://<ce>:<port>/arex/12345678:::ABCD`

We could use `pilotID` indeed, but this is not used in the other CEs.
And also, we could (we will) use the `AREXCE` to interact with jobs.

I like the idea of having CEs completely disconnected from any context, that we could use independently.
In such context, it would be the role of the caller to interpret the `jobID` returned. 

What do you think?

Reply:
Also fine as you put it.